include/aspect/material_model/utilities.h

```diff
@@ -223,25 +223,37 @@ namespace aspect
 
 
       /**
-       * For multicomponent material models: Given a vector of of compositional
-       * fields of length N, this function returns a vector of volume fractions
-       * of length N+1, corresponding to the volume fraction of a ``background
-       * material'' as the first entry, and volume fractions for each of the input
+       * For multicomponent material models: Given a vector of compositional
+       * field values of length N, this function returns a vector of fractions
+       * of length N+1, corresponding to the fraction of a ``background
+       * material'' as the first entry, and fractions for each of the input
        * fields as the following entries. The returned vector will sum to one.
        * If the sum of the compositional_fields is greater than
-       * one, we assume that there is no background mantle (i.e., that field value
+       * one, we assume that there is no background field (i.e., that field value
        * is zero). Otherwise, the difference between the sum of the compositional
-       * fields and 1.0 is assumed to be the amount of background mantle.
+       * fields and 1.0 is assumed to be the amount of the background field.
        * Optionally, one can input a component mask that determines which of the
        * compositional fields to use during the computation (e.g. because
-       * some fields contain non-volumetric quantities like strain,
+       * some fields contain unrelated quantities (like strain,
        * porosity, or trace elements). By default, all fields are included.
+       * This function makes no assumptions about the units of the
+       * compositional field values; for example, they could correspond to
+       * mass or volume fractions.
        */
       std::vector<double>
-      compute_volume_fractions(const std::vector<double> &compositional_fields,
-                               const ComponentMask &field_mask = ComponentMask());
-
+      compute_fractions_from_compositional_fields(const std::vector<double> &compositional_fields,
```

Comment:
I have a question and a comment here:
1. Would it make sense to name this `compute_composition_fractions`? This would be closer to the original name, and short, but carry the same meaning as far as I understand.
2. Please leave a copy of the old function for now, and deprecate it. The function implementation would then just call the new function. This is just because we usually do not immediately remove a function, but wait for at last one release before removing/renaming them. The old function would then look like this:
```
/**
* See compute_composition_fractions() for the documentation of this function.
* @deprecated: This function is deprecated. Please use compute_composition_fractions() instead.
       */
DEAL_II_DEPRECATED
      std::vector<double>
      compute_volume_fractions(const std::vector<double> &compositional_fields,
                               const ComponentMask &field_mask = ComponentMask());
```